implementations/rust/ockam/ockam_command/src/node/util.rs

```diff
@@ -144,3 +148,87 @@ pub async fn delete_embedded_node(cfg: &OckamConfig, name: &str) {
         let _ = tokio::fs::remove_dir_all(dir).await;
     }
 }
+
+pub fn delete_nodes_with_options(opts: CommandGlobalOpts, all: bool, force: bool) -> Result<()> {
+    if all {
```

Comment:
You could remove the `all` argument, as it has no effect. After that change, renaming the function to `delete_all_nodes` might be more explicit as to what it does.

Reply:
updates as per your comments